features/z-index.yml

```diff
@@ -0,0 +1,11 @@
+name: z-index
+description: The `z-index` CSS property allows you to order overlapping elements, with higher values in front of or on top of lower values.
+spec: https://drafts.csswg.org/css2/#z-index
+snapshot: ecmascript-5
+group: css
+compat_features:
+  - css.properties.z-index
+  - css.properties.z-index.auto
+  - css.properties.z-index.negative_values
+# Baseline low date of 2015 seems late, but is due to lack of Edge support until
+# then.
```

Comment:
I'm going to suggest deleting this comment otherwise, we're going to putting it _lots_ of places. 😅

Effectively, Baseline is undefined prior to the release of Edge 12 (since it's the youngest browser in the [core browser set](https://github.com/web-platform-dx/web-features/blob/main/docs/baseline.md#core-browser-set). It's possible that someday we'll do some sort of backtracking into IE data but for now this is just something we're living with.

See also: https://github.com/web-platform-dx/web-features/issues/623

Reply:
Excellent, thanks for the context!